src/client/datascience/jupyter/debuggerVariables.ts

```diff
@@ -127,7 +137,8 @@ export class DebuggerVariables extends DebugLocationTracker
         return results
             ? {
                   ...targetVariable,
-                  ...JSON.parse(results.result)
+                  ...JSON.parse(results.result),
+                  maximumRowChunkSize: 100
```

Comment:
Can we make this a constant somewhere `100`

Reply:
Touche, will fix